src/diff/props.js

```diff
@@ -35,13 +35,17 @@ function setStyle(style, key, value) {
 	if (key[0] === '-') {
 		style.setProperty(key, value);
 	}
+	else if (
+		typeof value === 'number' &&
+		IS_NON_DIMENSIONAL.test(key) === false
+	) {
+		style[key] = value + 'px';
+	}
+	else if (value == null) {
+		style[key] = '';
+	}
 	else {
-		style[key] =
-			typeof value === 'number' && IS_NON_DIMENSIONAL.test(key) === false
-				? value + 'px'
-				: value == null
-					? ''
-					: value;
+		style[key] = value;
```

Comment:
Terser compiles this into the same ternary we had manually written. I just found this to be much easier to read.

Reply:
Love this!